### PR TITLE
Fixed default consul tags and environment.

### PR DESCRIPTION
**Pull request by ArturKlajnerok**
Updated consul tags and environment in new service creation and in services that belong to @Wikia/services-team

### CODE REVIEW

**Comment by nmonterroso** [comment]
LGTM

**Comment by sqreek** [comment]
LGTM

File: service/example/src/main/java/com/wikia/exampleservice/guicemodule/ExampleModuleFactory.java [hunk 1]
```diff
@@ -9,8 +9,8 @@
 public class ExampleModuleFactory {
```
Comment (WikiaTech) [inline service/example/src/main/java/com/wikia/exampleservice/guicemodule/ExampleModuleFactory.java]: MAJOR Add a private constructor to hide the implicit public one. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Define and throw a dedicated exception instead of using a generic one.](https://github.com/Wikia/pandora/blob/c20f9e440541d9beb5685f39f055746ed817804f/service/example/src/main/java/com/wikia/exampleservice/guicemodule/ExampleModuleFactory.java#L31) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00112)

File: service/example/build.gradle [hunk 2]
```diff
@@ -18,14 +18,14 @@ deployments {
             instances = 2
         }
         environment {
-            CONSUL_TAGS = '["production", "prod"]'
-            ENVIRONMENT = "production"
+            CONSUL_TAGS = '["prod"]'
+            ENVIRONMENT = "prod"
```
Comment (jogura) [inline service/example/build.gradle]: I understand the change that the `CONSUL_TAGS` has on the service. What effect does changing the `ENVIRONMENT` variable have from `production` -> `prod`?

**Pull request merged**